h/views/api/groups.py

```diff
@@ -70,7 +71,7 @@ def remove_member(group, request):
     """Remove a member from the given group."""
 
     # Currently, we only support removing the requesting user
-    if request.matchdict.get('user') == 'me':
+    if request.matchdict.get('userid') == 'me':
```

Comment:
This parameter name has been changed here, but I don't believe it has been changed in the documentation for this endpoint. Can you please check?